srcpkgs/newsflash/template

```diff
@@ -0,0 +1,22 @@
+# Template file for 'newsflash'
+pkgname=newsflash
+version=1.4.2
+revision=1
+wrksrc="news_flash_gtk-$version"
+build_style=meson
+build_helper=rust
+hostmakedepends="cargo pkg-config glib-devel sqlite-devel gettext"
+makedepends="rust-std webkit2gtk-devel libhandy1-devel openssl-devel"
+short_desc="RSS feed reader that supports various web based services"
+maintainer="Michal Vasilek <michal@vasilek.cz>"
+license="GPL-3.0-only"
```

Comment:
It seems to be "or-later" https://gitlab.com/news-flash/news_flash_gtk/-/issues/222